apps/volontulo/views/pages.py

```diff
@@ -1,31 +1,34 @@
 # -*- coding: utf-8 -*-
-# pylint: disable=missing-docstring
+
+"""
+.. module:: pages
+"""
+
 from django.core.urlresolvers import reverse_lazy
 from django.shortcuts import redirect
 from django.views.generic.detail import DetailView
-from django.views.generic.edit import (
-    CreateView, UpdateView, DeleteView
-)
+from django.views.generic.edit import CreateView
+from django.views.generic.edit import DeleteView
+from django.views.generic.edit import UpdateView
```

Comment:
IMO one import per line is most readable way, only if you want to import  less than 3 or 5 modules. Otherwise it's going to look a bit odd. Anyway, I will pull new commit with imports done your way :)